docs/fdc3-glossary.md

```diff
@@ -1,4 +1,5 @@
 ---
```

Comment:
The final two terms are *not* in alpha order, but should be.

Reply:
Fixed, thanks